src/environments/environment.ts

```diff
@@ -3,6 +3,17 @@
 // `ng build --env=prod` then `environment.prod.ts` will be used instead.
 // The list of which env maps to which file can be found in `angular-cli.json`.
 
-export const environment = {
+export const environment = Object.freeze({
   production: false,
-};
+  config: {
+    apiEndpoint: 'http://localhost:8080/api/v1',
+    title: 'DEVELOPMENT - Red Hat iPaaS',
```

Comment:
I normally like to make it more obvious, e.g. banner or something on page but this is again, ok for now.